src/js/view/view.js

```diff
@@ -417,12 +417,16 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
-            if (provider && provider.getName().name.indexOf('flash') === 0) {
+            if (provider &&
```

Comment:
While we're at it, drop a comment explaining why these providers are excluded, or put this logic in a function (with a good name, of course)